blmath/geometry/primitives/plane.py

```diff
@@ -16,8 +16,8 @@ def __init__(self, point_on_plane, unit_normal):
 
         unit_normal = vx.normalize(unit_normal)
 
-        self._r0 = point_on_plane
-        self._n = unit_normal
+        self._r0 = np.asarray(point_on_plane)
```

Comment:
For those following along at home, moving the `np.array` from the getter to the constructor saved about half a million memory allocations. It's a 3 element array, but still those memory allocations added up.